src/Plivo/Resources/PhoneNumber/PhoneNumberInterface.php

```diff
@@ -110,7 +111,7 @@ public function buy($phoneNumber, $appId = null)
             );
         } else {
             throw new PlivoResponseException(
-                $responseContents['error'],
+                $responseContents['error_message'],
```

Comment:
@prabhav-plivo why has this been changed from the error to the error_message?

cc: @kapilp93